test/helpers/SessionAnswersHelperSpec.scala

```diff
@@ -663,4 +663,99 @@ class SessionAnswersHelperSpec extends SpecBase {
       }
     }
   }
+
+  "getContentForAgentsCheckYourAnswersPage" should {
+    "when the client planned to submit VAT return (so no cause Of LateSubmission chosen)" should {
+      "return a Seq[String, String, String] of answers" in {
+        val fakeRequestWithClientPresent = fakeRequest
+          .withSession(
+            SessionKeys.whoPlannedToSubmitVATReturn -> "client",
+            SessionKeys.causeOfLateSubmissionAgent -> "client"
```

Comment:
You wouldn't need to have `SessionKeys.causeOfLateSubmissionAgent` as that page isn't shown when the user select that the client had an issue.

Reply:
Done